### ISSUE TITLE
expose ingresses not picking up annotations

### ISSUE THREAD

**Issue by ryandawsonuk** [jenkins-x/exposecontroller#126]
I've got three different ingresses being created using exposecontroller (all in https://github.com/ryandawsonuk/environment-ferretmaze-staging/blob/e64d91e6c0072b1395736e0f0447bfe1064174ea/env/values.yaml#L32) and for each of these ingress when I do `kubectl describe ingress` the annotations section is empty
I may well have made a mistake in the yaml somewhere but I'm surprised that it's happening for all three.
It is working for a colleague of mine with almost exactly the same yaml. The only other differences being that I'm on AWS and he's on GCP and my jx is a little bit more up to date (jenkins x platform 0.0.1677).

The annotations are present on the services. But not on the ingress.

Struggling to think of what I can do to get to the bottom of it. Any suggestions would be appreciated.

**Comment by ryandawsonuk** [comment]
I've tried further expose config and [using exactly the same expose config](https://github.com/ryandawsonuk/environment-ferretmaze-staging/blob/6c61d9102e6ad108248c5cbfc4accebe9024236f/env/values.yaml#L100) that [I had working previously](https://github.com/ryandawsonuk/environment-ferretmaze-staging/blob/54102e0f275b902afaf085c8e9de81145fcce83b/env/values.yaml#L93) and which a [colleague of mine is using successfully ](https://github.com/miguelruizdev/environment-edgefir-staging/blob/16fb9c1683fd26fa7d3b4e8ff2c7b7bb8762669c/env/values.yaml#L93) but it is still not working for me (the Annotations section under `kubectl describe ingress` is blank). This colleague is on the same jx version as me but they are using GCP. Have there been any recent changes that could cause annotations to stop feeding through from services to ingress with expose specifically for AWS?

**Comment by ryandawsonuk** [comment]
I recently upgraded jx and now have:

![image](https://user-images.githubusercontent.com/22754674/43476409-709f6796-94f0-11e8-8a3c-66ceb4d6b168.png)

And tried this again with exposecontroller 2.3.69. It does populate a configmap I've set up with the clusterip and protocol of the service. But it doesn't set the exposeUrl or populate the annotations on the created Ingress. I suspect this is either related to my cluster being AWS or using wildcard DNS with route53 (as I said it did work for a colleague on GCP).
